app/templates/main/userProfile.html

```diff
@@ -105,12 +105,12 @@ <h5>You have not participated in any events.</h5>
   <!-- Start Program Eligibility Table -->
   <div class="accordion-item">
     <h3 class="accordion-header" id="headingThree">
-      {% set focus = "open" if visibleAccordion == "programTable" or volunteer.username != g.current_user.username else "collapsed" %}
+      {% set focus = "open" if visibleAccordion == "programTable" else "collapsed" %}
       <button class="accordion-button {{focus}}" type="button" data-bs-toggle="collapse" data-bs-target="#collapseThree" aria-expanded="false" aria-controls="collapseThree">
           Programs
       </button>
     </h3>
-    {% set show = "show" if visibleAccordion == "programTable" or volunteer.username != g.current_user.username else "" %}
+    {% set show = "show" if visibleAccordion == "programTable" else "" %}
```

Comment:
Why are you changing the accordion behavior?

Reply:
So from what I can tell, I only pull development a couple times since the last few commits and did not have any merge conflicts with development. I guess git is catching these at all and I just have an older version of the template. I'm not too sure.